<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculateForm</class>
 <widget class="QWidget" name="calculateForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(110, 122, 179, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>551</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(110, 122, 179, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableWidget" name="cTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>331</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 255);</string>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <row>
     <property name="text">
      <string>Billete</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>20</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>50</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>100</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>200</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>500</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1000</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total en billetes</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>NoItemFlags</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="flags">
      <set>NoItemFlags</set>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="9" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="10" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="cTotal_table">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 85, 255);</string>
    </property>
    <row>
     <property name="text">
      <string>CUP</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Total</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="cDelete_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(110, 122, 179, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>Borrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cCalculate_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(110, 122, 179, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 11pt &quot;Open Sans Semibold&quot;;

background-color: qlineargradient(spread:reflect, x1:0.035, y1:0, x2:0.806, y2:0.091, stop:0.20398 rgba(110, 122, 179, 0), stop:0.975124 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>Calcular Efectivo:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
